src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
```suggestion
public enum Color {
```
enum names are in singular